src/coreclr/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -3568,18 +3568,18 @@ public override Type[] GetGenericArguments()
         }
 
         [RequiresUnreferencedCode("If some of the generic arguments are annotated (either with DynamicallyAccessedMembersAttribute, or generic constraints), trimming can't validate that the requirements of those annotations are met.")]
-        public override Type MakeGenericType(Type[] instantiation)
+        public override Type MakeGenericType(Type[] typeArguments)
```

Comment:
You may want to fix it in the Mono variant too.

Reply:
Thanks - done